### PR TITLE
Add compliance and security response references

### PR DESCRIPTION
**Pull request by apburnes**
## Changes proposed in this pull request:
- Add reference link to Federalist contingency plan doc. [page](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/apb-federalist-contingency/docs/ops/contingency-plan/#federalist)
- Add link to TTS Hanbook's security response guide followed by Federalist. [page](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/apb-federalist-contingency/docs/ops/security-ir/#federalist)

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/apb-federalist-contingency)

## Security Considerations
- None

### CODE REVIEW

**Comment by apburnes** [comment]
@pburkholder is this enough?

File: _docs/ops/security-ir.md [hunk 1]
```diff
@@ -240,10 +242,14 @@ Guidelines for addressing Low-sev issues:
 - Responders should avoid service degradation unless stakeholders agree.
 - Sitreps should be sent approximately daily.
```
Comment (pburkholder) [inline _docs/ops/security-ir.md]: Let's  remove this. Federalist/Pages will just use the cloud.gov IR guide.

**Review by pburkholder — COMMENTED** [review]
Let's use the cloud.gov IR guide for Federalist, other changes look good.

**Comment by mheadd** [comment]
@apburnes Is this still needed?

**Comment by apburnes** [comment]
@mheadd I think it is good for a review now.

**Review by mheadd — APPROVED** [review]
🚀

**Pull request merged**